joomla/joomla-cms
# [4.0] ES6 multiselect.js

Pull Request for Issue # .

### Summary of Changes
Major refactor of this script, now uses js class and should be a lot faster.


### Testing Instructions
Check any of the list views (articles, menus, etc). Selecting either by:
- clicking on the checkboxes
- clicking anywhere in the table row (of course not on links or buttons :) )
- Check multiselecting by using shift


### Expected result



### Actual result



### Documentation Changes Required
